src/GuiBase/Viewer/Viewer.hpp

```diff
@@ -69,24 +71,12 @@ class RA_GUIBASE_API Viewer : public QWindow {
     /// create gizmos
     void createGizmoManager();
 
-    /**
-     * Make the OpenGL context associated with the viewer the current context.
-     */
-    void makeCurrent();
```

Comment:
Aren't these methods potentially useful for third party component ? 
For instance, to active the GL context before doing some GL operations ?

Reply:
they are in WindowQt, so available to third parties